### PR TITLE
Fix and enable integration tests

### PR DESCRIPTION
**Pull request by kismob**
This PR resolves https://github.com/linkedin/cruise-control/issues/1662

Hi @CCisGG, thank you for your patience. In the last couple days I was experimenting/debugging the tests. I came to a conclusion, that this is not a performance issue, it was something with the port of the Vertx app in the test cases.
This is a successful test run on Circle Ci (the workflow is a bit different, but it runs the ITs) : https://app.circleci.com/pipelines/github/kismob/cruise-control/48/workflows/d28e46a3-3ef7-43ca-964f-141f18bc3fe9/jobs/92

I hope it will finally resolve this issue.

### CODE REVIEW

**Comment by CCisGG** [comment]
Hi @kismob , thanks for the fix. Changes look good and we will merge it after the holiday break.

File: cruise-control/src/integrationTest/java/com/linkedin/kafka/cruisecontrol/ReplicaCapacityViolationIntegrationTest.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ public ReplicaCapacityViolationIntegrationTest(Boolean vertxEnabled) {
   public static Collection<Boolean> data() {
     // Skip the vertx enabled test for now, as it fails the integration test.
```
Comment (amuraru) [inline cruise-control/src/integrationTest/java/com/linkedin/kafka/cruisecontrol/ReplicaCapacityViolationIntegrationTest.java:62]: This comment can now be removed iiuc

**Review by viktorsomogyi — APPROVED** [review]

**Review by CCisGG — APPROVED** [review]

**Comment by CCisGG** [comment]
With this PR, the integration test passes in the latest run: https://app.circleci.com/pipelines/github/linkedin/cruise-control/2050/workflows/f944b996-50cd-4817-91d6-7a9d997bb2fa/jobs/5834. Thanks for the work!

**Comment by kismob** [comment]
Yeey! Thank you for your patience and all the reviews @CCisGG!

**Pull request merged**